x-pack/plugins/security_solution/cypress/integration/timelines/creation.spec.ts

```diff
@@ -71,10 +75,30 @@ describe('Timelines', (): void => {
     after(() => {
       closeTimeline();
     });
+    context('Privileges: CRUD', () => {
+      it('toggle create timeline ', () => {
+        createNewTimeline();
+        addNameAndDescriptionToTimeline(getTimeline());
+        cy.get(TIMELINE_PANEL).should('be.visible');
+      });
+    });
 
-    it('toggle create timeline ', () => {
-      createNewTimeline();
-      cy.get(TIMELINE_PANEL).should('be.visible');
+    context('Privileges: READ', () => {
+      before(() => {
+        login(ROLES.reader);
+        visit(OVERVIEW_URL, undefined, ROLES.reader);
+      });
+      it('should not be able to create/update timeline ', () => {
+        createNewTimeline();
+        cy.get(TIMELINE_PANEL).should('be.visible');
+        cy.get(EDIT_TIMELINE_BTN).should('be.disabled');
+        cy.get(EDIT_TIMELINE_BTN).first().trigger('mouseover', { force: true });
+        cy.get(EDIT_TIMELINE_TOOLTIP).should('be.visible');
+        cy.get(EDIT_TIMELINE_TOOLTIP).should(
+          'have.text',
+          'You can use Timeline to investigate events, but you do not have the required permissions to save timelines for future use. If you need to save timelines, contact your Kibana administrator.'
```

Comment:
nit: You could use the `i18n.translate` call here to make sure this test isn't failing when the translation change in the future

Reply:
will add it now.